README.md

```diff
@@ -113,25 +113,28 @@ Creating artifacts/abis/ to store compilation artifacts
 
 ### `deploy`
 
-> NOTICE: this method doesn't use an account, which will be deprecated very soon as StarkNet makes deployments from accounts mandatory.
+> NOTICE: Calling this method without the `--account` parameter doesn't use an account, which will be deprecated very soon as StarkNet makes deployments from accounts mandatory.
```

Comment:
The spirit of this is perfect and necessary, but I'd rephrase it a bit. Maybe something like:
```suggestion
> NOTICE: Calling this method will require an account (using the `--account` param) very soon as StarkNet will make deployments from accounts mandatory. 
```